META.md

```diff
@@ -76,6 +76,11 @@ meta exec "ghi label 'v1.2.0' -c '#bfdadc'"
 meta exec "ghi label 'v2.0.0' -c '#b94c47'"
 ```
 
+Alternate option if you are having issues with `ghi`:
+
+```
+meta exec "gh label create v2.2.0 -c '#537DB0'"
```

Comment:
Could you add tiny instructions on how to install GH?

Reply:
It is present at the top of the same file https://github.com/opensearch-project/opensearch-plugins/blob/7dc1ad8d3f85d8427a7988c709e20347dd730307/META.md#install-gh

Reply:
Haha thank you :)